src/medium-editor-phrase.js

```diff
@@ -221,7 +222,7 @@
       // select a text node where the original selection needs to be re-inserted
       selection.removeAllRanges();
       placeholderEl = ancestorPhraseParent.querySelector(placeholderSelector);
-      textNodePlaceholder = this.insertTextNodePlaceholderAfter(placeholderEl);
+      textNodePlaceholder = this.insertTextNodePlaceholder(placeholderEl, true);
```

Comment:
Like this change, only thing is when I look at this function call it's not immediately clear to me that the text node will be inserted after. 

Would kinda like to see a helper function like `insertTextNodePlaceholderAfter()` that would then make the call to this function as above. If you just call `insertTextNodePlaceholder(placeholderEl)` it'll consider this false via truthyness?